src/main/java/uk/gov/hmcts/cpp/audit/bff/client/ProgressionClient.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.cpp.audit.bff.client;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.hmcts.cpp.audit.bff.model.MaterialCaseResponse;
+
+@Component
+public class ProgressionClient {
+
+    private static final String HEADER_USER = "CJSCPPUID";
+    private static final String HEADER_CORR = "CPPCLIENTCORRELATIONID";
+
+    private final RestTemplate restTemplate;
+    private final String progressionServiceUrl;
+    private final String headerUserId;
+
+    public ProgressionClient(RestTemplate restTemplate,
+                             @Value("${progression.service.baseUrl}") String progressionServiceUrl,
+                             @Value("${progression.service.cjscppuid}") String headerUserId) {
+        this.restTemplate = restTemplate;
+        this.progressionServiceUrl = progressionServiceUrl;
+        this.headerUserId = headerUserId;
+    }
+
+    private HttpEntity<?> createEntityWithHeaders(String correlationId) {
+        HttpHeaders headers = new HttpHeaders();
+        headers.set(HEADER_USER, headerUserId);
+        headers.set(HEADER_CORR, correlationId);
+        return new HttpEntity<>(headers);
+    }
+
+    public MaterialCaseResponse getMaterialCases(String materialIds, String correlationId) {
+        String url = UriComponentsBuilder.fromUriString(progressionServiceUrl)
+            .path("/progression-query-api/query/api/rest/progression/materials/cases")
+            .queryParam("materialIds", materialIds)
+            .toUriString();
+
+        var response = restTemplate.exchange(url, HttpMethod.GET,
+                                             createEntityWithHeaders(correlationId), MaterialCaseResponse.class);
```

Comment:
I think there is an issue here. The progression API returns the results as a list under a `materialIds` property. https://github.com/hmcts/cpp-context-progression/pull/1444/files#diff-6ec764573abd09f17b62fdc845b31a5ec664ebef2f074bf8c151e1ddc93a8e78